config/settings.yml

```diff
@@ -14,12 +14,10 @@ hmac_secret: ~
 filesystems:
   purl_root: "/purl/document_cache"
   stacks_root: "/stacks"
-  stacks_content_addressable: /stacks/content_addressable
```

Comment:
Are we not using this path for versioned objects anymore?  I thought we agreed (with Julian) that we would put all of the versioned objects in a separate root.

Reply:
I proposed putting all of the versioned objects in a separate root. I don't think Julian was a hard no on it, but preferred that it not be in a separate root (so that the linked files would all be close together and definitely on the same file system). I didn't see any compelling reason not to take that approach.